menu.js

```diff
@@ -0,0 +1,19 @@
+const icon = document.querySelector('#icon');
+icon.addEventListener('click', () => {
+  const container = document.querySelector('.container');
+  const nav = document.querySelector('nav');
+  const span = document.createElement('span');
+  span.innerHTML += '<span>&times;</span>';
+  span.classList.add('close');
+  nav.appendChild(span);
+  if (nav.style.display === 'inline-block') {
+    nav.style.display = 'none';
+  } else {
+    nav.style.display = 'block';
+    nav.classList.add('nav-style');
+    span.addEventListener('click', () => {
+      container.removeChild(nav);
+    });
+    document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => { container.removeChild(nav); }));
+  }
```

Comment:
- [ ] Also, I think it would be better to blur the background to match your [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546) as much as possible 👌

> Hint: there is a property in CSS called [filter](https://www.w3schools.com/cssref/css3_pr_filter.asp) `filter: blur(3px)` you can add it to the `body` or `section-1` class when you open the menu and remove it when you close the menu 🤝